dev-tools/scripts/releaseWizard.yaml

```diff
@@ -636,7 +636,7 @@ groups:
       Ask on dev@ for input. Ideally the timing of this request mostly coincides with the
       release branch creation. It's a good idea to remind the devs of this later in the release too.
 
-      NOTE: Do not add every single JIRA issue, but distill the Release note into important changes!
+      NOTE: Do not add every single JIRA issue, but distill the Release notes into important changes!
```

Comment:
capitalization: I think this should either be `release notes` or `Release Notes` (not sure how it is referred to elsewhere, most important is consistency.)

Reply:
agreed, used 'release notes' so it's same as above.